src/test/java/com/iota/iri/profiling/RocksDbSizeProfiling.java

```diff
@@ -0,0 +1,137 @@
+package com.iota.iri.profiling;
+
+import com.iota.iri.conf.BaseIotaConfig;
+import com.iota.iri.controllers.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.iota.iri.controllers'](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4804250)

Reply:
Need to fix codacy import repoting